### DISCUSSION TITLE
How do I animate things with just theme-ui?

### DISCUSSION THREAD

**Discussion by wahidshafique** [system-ui/theme-ui#1306] [Q&A]
I was wondering how I can do this without importing another library to expose things like `keyframes`. Is there a sane way to put animations inside of the `sx` prop using just theme-ui?

**Comment by atanasster** [comment]
@wahidshafique , you could use the Spinner component as an example with emotion https://github.com/system-ui/theme-ui/blob/master/packages/components/src/Spinner.js

**Comment by hasparus** [comment]
Okay, we should really reexport these keyframes. Yup. `@emotion/react` is already in your node_modules, because we depend on it. `keyframes` would be the simplest/default way of animation if you're using Theme UI for styling.

**Reply by lachlanjc** [reply]
I actually had an idea for a custom Theme UI animation hook last week, which I’m working on :)
